taipy/gui/viselements.json

```diff
@@ -504,7 +504,7 @@
                     },
                     {
                         "name": "animation_data",
-                        "type": "dynamic(dict[str, Any])",
+                        "type": "chart.data",
```

Comment:
This should be 'dynamic(Any)': it is used when generating the Page Builder API, and must be a valid Python type.